src/verilog/verilog_typecheck.cpp

```diff
@@ -951,6 +955,61 @@ void verilog_typecheckt::convert_function_call_or_task_enable(
 
 /*******************************************************************\
 
+Function: verilog_typecheckt::convert_paramter_override
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+void verilog_typecheckt::convert_parameter_override(
+  const verilog_parameter_overridet &module_item)
+{
+  for(auto &assignment : module_item.assignments())
+  {
+    auto lhs = to_binary_expr(assignment).lhs();
+    auto rhs = to_binary_expr(assignment).rhs();
```

Comment:
For me, the use of `auto` made things more difficult: I believe this will be `exprt`, and neither a reference nor will it be `const`. I'd suggest to use `const exprt &lhs`, `const exprt &rhs`.

Reply:
Now exprt, with a comment that a copy is made.